values.yaml

```diff
@@ -5,6 +5,8 @@
 ## Docker image settings, applied to all StackStorm pods
 ##
 image:
+
+  utilityImage: ""
```

Comment:
This needs a doc string and a comment showing an example.

Reply:
added a doc string